### PR TITLE
Create dom-range.json

### PR DESCRIPTION
**Pull request by dandv**
https://developer.mozilla.org/en-US/docs/Web/API/Range lists a detailed feature compatibility table. That is to be integrated in the JSON at a later time.

### CODE REVIEW

File: features-json/dom-range.json [hunk 1]
```diff
@@ -0,0 +1,228 @@
...
+  "spec":"http://www.w3.org/TR/DOM-Level-2-Traversal-Range/ranges.html",
+  "status":"rec",
+  "links":[
+    {
+      "url":"https://developer.mozilla.org/en-US/docs/Web/API/Range",
+      "title":"MDN"
+    },
+    {
+      "url":"http://www.quirksmode.org/dom/range_intro.html",
+      "title":"QuirksMode"
```
Comment (cvrebert) [inline features-json/dom-range.json]: Missing closing curly brace

File: features-json/dom-range.json [hunk 2]
```diff
@@ -0,0 +1,229 @@
...
+    }
+  ],
+  "bugs":[],
+  "categories":[
+    "JS API",
+    "DOM:
```
Comment (cvrebert) [inline features-json/dom-range.json]: Missing closing quote

File: features-json/dom-range.json [hunk 3]
```diff
@@ -0,0 +1,229 @@
...
+      "31":"y",
+      "32":"y",
+      "33":"y",
+      "34":"y",
+      "35":"y",
+      "36":"y",
+      "37":"y",
+      "38":"y",
+      "39":"y",
+      "40":"Y",
```
Comment (cvrebert) [inline features-json/dom-range.json]: Must be a lowercase y. See https://travis-ci.org/Fyrd/caniuse/builds/69074589

**Comment by Fyrd** [comment]
Thanks!

**Pull request merged**